src/crypto/rsa.rs

```diff
@@ -4,8 +4,10 @@
 
 //! RSA, a public-key encryption algorithm.
 
+use crate::crypto::sig;
+
 #[cfg(doc)]
-use std::convert::Infallible;
+use sig::*;
```

Comment:
I know there isn't really a policy, but can we avoid using wildcard imports?

Reply:
Mm, we could. I was making an exception for documentation specifically, but I think for lack of a specific policy I can explode it into a braced `use`.